documentation/spring-boot-starter.html.haml

```diff
@@ -0,0 +1,175 @@
+---
+layout: base
+---
+
+%h1
+  Spring Boot Starter <span class="label label-important">since 2.3.0</span>
+
+%h2
+  Getting Started
+
+%p
+  Add the Togglz Spring Boot Starter dependency to your project:
+
+%pre(class="prettyprint lang-xml")
+  :escaped
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-spring-boot-starter</artifactId>
+      <version>#{site.stableVersion}</version>
+    </dependency>
+
+%p
+  Or when using Gradle simply:
+  
+%pre(class="prettyprint")
+  :escaped
+    compile("org.togglz:togglz-spring-boot-starter:#{site.stableVersion}")
+
+%p
+  The starter will not only add the <code>togglz-core</code> and <code>togglz-spring-core</code> modules to your project,
+  it will also enable the auto configuration of the Togglz configration.
+
+%p
+  Optionally add the <code>togglz-console</code>, <code>togglz-spring-security</code>, <code>togglz-junit</code> and <code>thymeleaf-extras-togglz</code> modules:
+
+%pre(class="prettyprint lang-xml")
+  :escaped
+    <!-- Togglz Admin Console (optional) -->
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-console</artifactId>
+      <version>#{site.stableVersion}</version>
+    </dependency>
+    
+    <!-- Togglz Spring Security (optional) -->
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-spring-security</artifactId>
+      <version>#{site.stableVersion}</version>
+    </dependency>
+    
+    <!-- Togglz JUnit Testing Support (optional) -->
+    <dependency>
+      <groupId>org.togglz</groupId>
+      <artifactId>togglz-junit</artifactId>
+      <version>#{site.stableVersion}</version>
+      <scope>test</scope>
+    </dependency>
+    
+    <!-- Thymeleaf Togglz Dialect (optional) -->
+    <dependency>
+      <groupId>com.github.heneke.thymeleaf</groupId>
+      <artifactId>thymeleaf-extras-togglz</artifactId>
+      <version>1.0.1.RELEASE</version>
+      <scope>test</scope>
```

Comment:
Good catch! You are absolutely right this very awkward :-).
I will remove the test scope for the thymeleaf dependency.